src/aggregated_non_cooperative.jl

```diff
@@ -318,6 +320,7 @@ function to_objective_callback_by_subgroup(::AbstractGroupANC, ECModel::Abstract
 
                 gen_data = ecm_copy.gen_data
                 market_data = ecm_copy.market_data
+                u_standard = first(keys(users_data))
```

Comment:
This and the following still needs revision

Reply:
Resolved